lms/envs/common.py

```diff
@@ -127,20 +198,42 @@
 
     'DISABLE_LOGIN_BUTTON': False,  # used in systems where login is automatic, eg MIT SSL
 
-    # Toggles OAuth2 authentication provider
+    # .. toggle_name: ENABLE_OAUTH2_PROVIDER
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Enable this feature to turn this Open edX platform in an OAuth2 authentication provider.
+    #   This is necessary to enable some other features, such as the REST API for the mobile application.
+    # .. toggle_category: admin
+    # .. toggle_use_cases: open_edx
```

Comment:
I don't believe the platform is usable anymore without OAuth2 enabled. If others agree, can we change this use-case to be a `temporary` one so we can mark this toggle for deletion?

Reply:
:ok_hand: I'll flag this as `monitored_rollout` for now and will update all `toggle_use_cases` fields once we land the code-annotation PR that simplifies the annoation format.